feature/google/src/bc5SupportDev/java/com/revenuecat/purchases/google/billingClientParamBuilders.kt

```diff
@@ -0,0 +1,12 @@
+package com.revenuecat.purchases.google
+
+import com.android.billingclient.api.QueryPurchaseHistoryParams
+import com.android.billingclient.api.QueryPurchasesParams
+
+fun String.buildQueryPurchaseHistoryParams(): QueryPurchaseHistoryParams {
+    return QueryPurchaseHistoryParams.newBuilder().setProductType(this).build()
```

Comment:
if we check that the string is one of `BillingClient.ProductType`, this has to be nullable. leaning towards making it a QueryPurchaseHistoryParams extension that takes in the string so we can use the type annotation.

@vegaro